integration_tests/__init__.py

```diff
@@ -104,3 +106,10 @@ def logout(self):
         expected = ('Clearing credentials for Ubuntu One SSO.\n'
                     'Credentials cleared.\n')
         self.assertEqual(expected, output)
+
+    def register_name(self, name):
+        process = pexpect.spawn(self.snapcraft_command,
+                                ['register-name', name])
```

Comment:
Do we need to call the command register-name? Wouldn't it be clear enough to call it just register?